src/overlays/actors/ovl_Obj_Wind/z_obj_wind.c

```diff
@@ -28,21 +29,144 @@ ActorInit Obj_Wind_InitVars = {
     (ActorFunc)ObjWind_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80B24480[] = {
+static InitChainEntry sInitChain[] = {
     ICHAIN_F32(uncullZoneForward, 4000, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneScale, 4000, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneDownward, 4000, ICHAIN_STOP),
 };
 
-#endif
+void ObjWind_Init(Actor* thisx, PlayState* play) {
+    ObjWind* this = (ObjWind*)thisx;
+    s32 pad;
+    WaterBox* waterbox;
+    f32 ySurface;
 
-extern InitChainEntry D_80B24480[];
+    Actor_ProcessInitChain(&this->actor, sInitChain);
+    if (WaterBox_GetSurface1(play, &play->colCtx, this->actor.world.pos.x, this->actor.world.pos.z, &ySurface,
+                             &waterbox) &&
+        (this->actor.world.pos.y < ySurface)) {
+        this->underWater = 1;
```

Comment:
```suggestion
        this->isUnderWater = true;
```
Weird that this isn't used